### DISCUSSION TITLE
Flutter (Dart) with NvChad

### DISCUSSION THREAD

**Discussion by fegyi001** [NvChad/NvChad#1692] [Q&A] [answered]
Hello,

I'd like to know how should I set my custom/ folder to enable Flutter.
Has anyone done that before?

Thanks in advance!

**Comment by siduck** [comment, marked as answer]
@fegyi001 hi, just setup flutter-tools.nvim plugin

**Reply by fegyi001** [reply]
awesome, it works, thanks!
